### PR TITLE
Create LIT7052Qene.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Qene type still needed.

### CODE REVIEW

File: new/LIT7052Qene.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55rb-c. Division into verses is indicated with
+                    <foreign xml:lang="gez">።</foreign>. Division into verses is indicated with <foreign xml:lang="gez">።</foreign>. 
+                    The end of the poem is indicated with <foreign xml:lang="gez">። ።</foreign>.</note>
+                <ab>
+                    <l n="1">ኢሀሎ፡ በዓለም፡ ዲበ፡ ሰረገላ፡ እንዘ፡ በዘባኑ፡ ለአድግ፡ ነባቤ፡ ነገር፡ ሠናይ።</l>
+                    <l n="2">አመ፡ ተዘከረ፡ ይርግም፡ ቤተ፡ ያዕቆብ፡ ኀሩይ።</l>
+                    <l n="3">ወኤልያስ፡ እንዘ፡ ሀሎ፡ ዲበ፡ ሰረገላ፡ ዘእሳት፡ ውዑይ።</l>
+                    <l n="4">ኢክህለ፡ ይሴሰይ።</l>
+                    <l n="5">ተጽዕኖተ፡ ዕድገት፡ መዓረ፡ ወተ<add place="above">ጽ</add>ዕኖ፡ ሰዋል፡ ፍሬ፡ ሠናይ።</l>
```
Comment (DenisNosnitsin1970) [inline new/LIT7052Qene.xml:65]: I think here it is ዕዋል, not ሰዋል. (the meaning "young animal" fits)

File: new/LIT7052Qene.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7052Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ኢሀሎ፡ በዓለም፡ ዲበ፡ ሰረገላ፡ እንዘ፡ በዘባኑ፡ ለአድግ፡ ነባቤ፡ ነገር፡ ሠናይ።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾI-hallo ba-ʿālam diba saragalā ʾǝnza ba-zabānu la-ʾadg nabābe nagar śannāy.</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT7052Qene.xml:9]: Thank you! I would say: In the world, he is not on the chariot but on the back of the donkey, the speaker of the good things. ...

saragalla with ll

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT7052Qene.xml [hunk 3]
```diff
@@ -6,8 +6,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <fileDesc>
             <titleStmt>
                 <title xml:lang="gez" xml:id="t1">ኢሀሎ፡ በዓለም፡ ዲበ፡ ሰረገላ፡ እንዘ፡ በዘባኑ፡ ለአድግ፡ ነባቤ፡ ነገር፡ ሠናይ።</title>
-                <title xml:lang="gez" corresp="#t1" type="normalized">ʾI-hallo ba-ʿālam diba saragalā ʾǝnza ba-zabānu la-ʾadg nabābe nagar śannāy.</title>
-                <title xml:lang="en" corresp="#t1">There is nothing in the world on the chariot, since the speaker of the good news is on the back of a donkey. ...</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾI-hallo ba-ʿālam diba saragallā ʾǝnza ba-zabānu la-ʾadg nabābe nagar śannāy.</title>
+                <title xml:lang="en" corresp="#t1">In the world, he is not on the chariot, but on the back of the donkey - the speaker of the good things. ...</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT7052Qene.xml:10]: I would use comma after donkey (open the possibility of interpretation who the speaker is, He or donkey :) ). I am not fully certain that I fully understand it ()

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT7052Qene.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT7052Qene.xml]: I assume, that "kwellekkemu type" is [Kʷǝllǝkǝmu gǝʿǝz](https://betamasaheft.eu/authority-files/list?keyword=Kwellekomu) in the list of Qene-types on the Bm-website.

**Review by DenisNosnitsin1970 — APPROVED** [review]

File: new/LIT7052Qene.xml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne of the Kʷǝllǝkǝmu gǝʿǝz-type. Type and author are not indicated in the manuscript.</p>
```
Comment (karljonaskarlsson) [inline new/LIT7052Qene.xml:39]: I would transcribe "gǝʿz", as we are not talking about the language here

File: new/LIT7052Qene.xml [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55rb-c. Division into verses is indicated with
+                    <foreign xml:lang="gez">።</foreign>. The end of the poem is indicated with <foreign xml:lang="gez">። ።</foreign>.</note>
+                <ab>
+                    <l n="1">ኢሀሎ፡ በዓለም፡ ዲበ፡ ሰረገላ፡ እንዘ፡ በዘባኑ፡ ለአድግ፡ ነባቤ፡ ነገር፡ ሠናይ።</l>
+                    <l n="2">አመ፡ ተዘከረ፡ ይርግም፡ ቤተ፡ ያዕቆብ፡ ኀሩይ።</l>
```
Comment (karljonaskarlsson) [inline new/LIT7052Qene.xml:64]: ኅሩይ፡ (cf. ኀበ፡ and ማኅቶተ፡ higher up in the same column)

File: new/LIT7052Qene.xml [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+                    <l n="3">ወኤልያስ፡ እንዘ፡ ሀሎ፡ ዲበ፡ ሰረገላ፡ ዘእሳት፡ ውዑይ።</l>
+                    <l n="4">ኢክህለ፡ ይሴሰይ።</l>
+                    <l n="5">ተጽዕኖተ፡ ዕድገት፡ መዓረ፡ ወተ<add place="above">ጽ</add>ዕኖ፡ ዕዋል፡ ፍሬ፡ ሠናይ።</l>
```
Comment (karljonaskarlsson) [inline new/LIT7052Qene.xml:67]: ዕድግት, maybe?

File: new/LIT7052Qene.xml (hunk 4 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT7052Qene.xml:39]: It is transcribed like this in Bm. I want to keep consistency in the transcription: 

Screenshot 2024_Kwellekemu geez

Comment (karljonaskarlsson) [reply new/LIT7052Qene.xml:39]: If that case, I think it would be better to change the name of the qene form and use "gǝʿz" everywhere

**Pull request merged**